components/ProductMatrix.vue

```diff
@@ -40,17 +45,52 @@ export default Vue.extend({
       }
     }
   },
-  computed: mapGetters({
-    cards: 'cards/getCards',
-    loading: 'products/isLoading'
-  }),
+  computed: {
+    columns(): number {
+      const columns = Math.floor(this.windowWidth / 152)
```

Comment:
Ok, this is a merge of another code base, but checking the card width or (temporary?) using a variable called ` cardWidth = 152` should be a better choice.

Reply:
Good point 👍